git/gitconfig.symlink

```diff
@@ -1,53 +1,63 @@
-# local/private config pulled from include
+######## local overrides of `git` identity (author name & email) ########
+# https://git-scm.com/docs/git-config#_includes
 [include]
 	path = ~/.gitconfig.local
-[hub]
-	protocol = ssh
-[color]
-	ui = always
-	diff = auto
-	status = auto
-	branch = auto
+######## alphabetical list of config options ########
+[color "status"]
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-colorstatusltslotgt
+	added = 045
+	branch = 135
+	changed = 212
+	localBranch = 135
+	remoteBranch = 159
+	untracked = 004
 [core]
-	excludesfile = ~/.gitignore
+	# maintains original EOL characters on checkout, normalizes CLRF -> LF endings
+	# on commit
+	# https://stackoverflow.com/a/4425433
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreautocrlf
+	autocrlf = input
+	# marks `~/.gitignore` (if it exists) as a global ignore file applying to all
+	# `git` repos throughout your machine (unrelated to the `.gitignore` at the
+	# root of this repo - `$DOTFILES/.gitignore`)
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreexcludesFile
+	excludesFile = ~/.gitignore
```

Comment:
@sbarbeau-od thoughts on this documentation? any suggestions for improvement?

Reply:
LGTM :) Thanks